tla-types/src/main/scala/at/forsyte/apalache/tla/typecomp/signatures/ApalacheInternalOperSignatures.scala

```diff
@@ -46,6 +49,9 @@ object ApalacheInternalOperSignatures {
           },
       )
 
+    // (t, Set(t)) => Bool
+    val storeNotInSetSig = signatureMapEntry(storeNotInSet, { case Seq(t, SetT1(tt)) if t == tt => BoolT1 })
```

Comment:
So this one is just sets?

Reply:
Yes, this one should only touch sets. I will fix the tests.

Reply:
Ok, this will be a good experiment to see if the testing framework is intuitive.